arktype.io/src/pages/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { useColorMode } from "@docusaurus/theme-common"
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext"
+import { ThemeProvider, Typography } from "@mui/material"
+import Layout from "@theme/Layout"
+import { motion } from "framer-motion"
+import React from "react"
+import { StackBlitzDemo } from "../../docs/demos"
+import { getTheme } from "../components"
+import { Boat } from "../components/svg/boat"
+import { JS } from "../components/svg/js"
+import { TS } from "../components/svg/ts"
+
+const Contents = () => {
+    const { siteConfig } = useDocusaurusContext()
+    const { colorMode } = useColorMode()
+    const isDark = colorMode === "dark"
+    return (
+        <ThemeProvider theme={getTheme({ isDark })}>
+            <Logo title={siteConfig.title} tagline={siteConfig.tagline} />
+            <main>
+                <StackBlitzDemo embedId="type" />
+            </main>
+        </ThemeProvider>
+    )
+}
+
+// eslint-disable-next-line import/no-default-export
+export default () => {
+    const { siteConfig } = useDocusaurusContext()
+    return (
+        <Layout title={siteConfig.title} description={siteConfig.tagline}>
+            <Contents />
+        </Layout>
+    )
+}
+
+// eslint-disable-next-line max-lines-per-function
```

Comment:
Instead of adding comment here, update .eslintrc.cjs rule referencing redo.dev to also reference arktype.io